e2e/backup_test.go

```diff
@@ -33,28 +34,34 @@ var _ = Context("backup", func() {
 	var restorePoint time.Time
 
 	It("should create a bucket", func() {
-		kubectlSafe(nil, "run", "--command", "make-bucket", "--image=moco-backup:dev", "--",
-			"s3cmd", "--host=minio.default.svc:9000", "--host-bucket=minio.default.svc:9000", "--no-ssl",
-			"--access_key=minioadmin", "--secret_key=minioadmin", "mb", "s3://moco")
+		kubectlSafe([]byte(makeBucketYAML), "apply", "-f", "-")
+		Eventually(func(g Gomega) {
+			out, err := kubectl(nil, "get", "jobs", "make-bucket", "-o", "json")
+			g.Expect(err).NotTo(HaveOccurred())
+			job := &batchv1.Job{}
+			err = json.Unmarshal(out, job)
+			g.Expect(err).NotTo(HaveOccurred())
+			condComplete, err := getJobCondition(job, batchv1.JobComplete)
+			g.Expect(err).NotTo(HaveOccurred())
+			g.Expect(condComplete.Status).To(Equal(corev1.ConditionTrue), "make-bucket has not been finished")
+		}).Should(Succeed())
 	})
 
 	It("should construct a source cluster", func() {
 		kubectlSafe(fillTemplate(backupYAML), "apply", "-f", "-")
-		Eventually(func() error {
+		secjson := kubectlSafe(nil, "get", "secret", "-o", "json", "minio-cert")
+		sec := &corev1.Secret{}
+		json.Unmarshal(secjson, sec)
+		sec.SetNamespace("backup")
+		secjson, err := json.Marshal(sec)
+		Expect(err).NotTo(HaveOccurred())
+		kubectlSafe(secjson, "apply", "-f", "-")
```

Comment:
@kmdkuk 
Is this secret necessary for `backup_test.go`?

Reply:
@morimoto-cybozu 
Yes, it is unnecessary.
Thank you.